app/src/main/java/org/simple/clinic/textInputdatepicker/TextInputDatePickerModel.kt

```diff
@@ -0,0 +1,12 @@
+package org.simple.clinic.textInputdatepicker
```

Comment:
Instead of calling it `Extend BaseBottomSheet in TextInputDatePicker` we can rename this to `Create TextInpuDatePicker` or something like that. Because we are creating a new sheet and not just extending the `BaseBottomSheet` for already existing screen right?. 

Also just an FYI, usually we create layout files and class files in a single commit either at the start or at the end when wiring up the loop with UI.